services/welding-machine-defect-detection-model-service/app/services/predict_service.py

```
@@ -0,0 +1,51 @@
+import numpy as np
+import numpy as np
```

Ruff (0.12.2)

Redefinition of unused `np` from line 1

Remove definition: `np`

(F811)